sql_jobs/hourly/skill_ranks.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE IF NOT EXISTS skill_ranks (
+    id bigint,
+    normalized_name varchar(128),
+    name varchar(128),
+    total_count numeric
+);
+
+CREATE TABLE skill_ranks_tmp
+AS SELECT * FROM skill_ranks;
+
+DELETE FROM skill_ranks_tmp;
```

Comment:
Apparently truncate is slightly faster when deleting with no clause like this, but speed different is very likely insignificant.

Reply:
super interesting :). I definitely look forward to when we need to thinka bout speed more. :)